lookup.go

```diff
@@ -93,8 +94,10 @@ func (dht *IpfsDHT) GetClosestPeers(ctx context.Context, key string) (<-chan pee
 	go func() {
 		defer close(out)
 		defer e.Done()
+		timedCtx, cancel := context.WithTimeout(ctx, time.Minute)
+		defer cancel()
 		// run it!
-		res, err := query.Run(ctx, tablepeers)
+		res, err := query.Run(timedCtx, tablepeers)
```

Comment:
@Stebalien is this what you're talking about?

Reply:
Exactly! If we want to get fancy, we could even get the deadline from the context and subtract 1(?) second (we should be connected to the closest peers when this command returns.

Yay for arbitrary constants!